iBudget/templates/emails/change_password_link.html

```diff
@@ -0,0 +1,14 @@
+<h1>Hello
+    <i>
```

Comment:
It is better to use less indentation in email templates. Sometimes these indentation will appear in real email.